speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/ConfigEnums.h

```diff
@@ -1,16 +1,19 @@
 #pragma once
 
+/// @brief The different tasks that can be performed
 typedef unsigned int Task;
 
 #define TASK_NO_TASK        0
 #define TASK_LANE_KEEPING   1
 #define TASK_SPEED_CONTROL  2
 
+/// @brief The different indicators that can be shown
```

Comment:
I'd reword it to something closer to "The different ways a user can be indicated about an intervention" since "shown" doesn't apply to auditory indicators